### PR TITLE
Tahoe registration API - added test for support methods

### PR DESCRIPTION
**Pull request by johnbaldwin**

### CODE REVIEW

File: openedx/core/djangoapps/appsembler/api/tests/test_registration_viewset.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+from openedx.core.djangoapps.appsembler.api.v1.views import RegistrationViewSet
+
+
+@ddt.ddt
+class RegistrationViewSetMethodTestCase(TestCase):
+    def setUp(self):
+        self.reg = RegistrationViewSet()
+
+    @ddt.data(True, False, 'True', 'False', 'true', 'false')
```
Comment (OmarIthawi) [inline openedx/core/djangoapps/appsembler/api/tests/test_registration_viewset.py]: For simple scalar data, I don't really trust `@ddt.data`.

File: openedx/core/djangoapps/appsembler/api/tests/test_registration_viewset.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    def test_normalize_bool_param(self, unnormalized):
+        expected_map = {
```
Comment (OmarIthawi) [inline openedx/core/djangoapps/appsembler/api/tests/test_registration_viewset.py]: Same goes for using `dict`. Type conversion is very likely.

**Comment by johnbaldwin** [comment]
@OmarIthawi I added a test for the `_normalize_bool_param` support method

File: openedx/core/djangoapps/appsembler/api/tests/test_registration_viewset.py [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+            True: True,
+            False: False,
+            'True': True,
+            'False': False,
+            'true': True,
+            'false': False
+        }
+        normalized = self.reg._normalize_bool_param(unnormalized)
```
Comment (OmarIthawi) [inline openedx/core/djangoapps/appsembler/api/tests/test_registration_viewset.py]: Please opt for old-fashioned bare `assert` such as:

```python
assert self.reg._normalize_bool_param(True)
assert self.reg._normalize_bool_param('True')
assert self.reg._normalize_bool_param('true')

assert not self.reg._normalize_bool_param('false')
assert not self.reg._normalize_bool_param('False')
assert not self.reg._normalize_bool_param(False)
```

That :point_up: can't go wrong.

**Pull request merged**